### PR TITLE
Fix expression tree tests

### PR DESCRIPTION
**Pull request by sgeor255**

### CODE REVIEW

File: test/exprtest/blas1_axpy_copy_test.cpp [hunk 1]
```diff
@@ -23,6 +23,7 @@
  *
  **************************************************************************/
 #include "blas_test.hpp"
+#include "blas_test_macros.hpp"
```
Comment (Rbiessy) [inline test/exprtest/blas1_axpy_copy_test.cpp:26]: The other tests don't include this file as it's already included by `blas_test.hpp`. Can we remove them?

**Review by Rbiessy — COMMENTED** [review]
Thanks for the fix!

File: test/exprtest/blas1_axpy_copy_test.cpp (hunk 1 above)
Comment (sgeor255) [reply test/exprtest/blas1_axpy_copy_test.cpp]: My bad, removed it.

**Review by Rbiessy — APPROVED** [review]

**Review by KumudhaN — APPROVED** [review]
LGTM. Thanks for fixing this!

**Pull request merged**